### PR TITLE
Issue-166 Hamburger back button changes

### LINKED ISSUE #166: Back button hidden in hamburger in mobile view
Current
image

Desired 
image

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #166

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.live/
- After: https://issue-166--nedbank--hlxsites.hlx.live/

### CODE REVIEW

**Comment by jindaliiita** [comment]
@sdmcraft This PR contains changes wrt to back button on smaller screen for hamburger.
There are many other styling issues which we discussed related to anchor tags etc.
I tried unsetting anchor tag properties but hamburger is also picking up properties from body tag of styles.less.
Will discuss this again in standup.

File: blocks/header/nav-utils.js [hunk 1]
```diff
@@ -97,6 +103,18 @@ export async function loadNavTools() {
     document.body.appendChild(externalMarkup);
 
     const hamburgerModal = doc.querySelector('.nbd-hamburger-menu-wrapper');
+    const hamburgerNavWrapper = doc.querySelector('.nbd-navbar-desktop-wrapper');
+
+    if (window.screen.width < 1025 && hamburgerNavWrapper) {
```
Comment (sdmcraft) [inline blocks/header/nav-utils.js:108]: Can we add the `navWrapper` in all cases but not display it for screen width < 1025 through css media query?

**Comment by sdmcraft** [comment]
> @sdmcraft This PR contains changes wrt to back button on smaller screen for hamburger.
> There are many other styling issues which we discussed related to anchor tags etc.
> I tried unsetting anchor tag properties but hamburger is also picking up properties from body tag of styles.less.
> Will discuss this again in standup.

Would using `all: unset` or doing something similar in the copied over clientlibs styles at the root level help? See https://developer.mozilla.org/en-US/docs/Web/CSS/all

**Comment by jindaliiita** [comment]
> > @sdmcraft This PR contains changes wrt to back button on smaller screen for hamburger.
> > There are many other styling issues which we discussed related to anchor tags etc.
> > I tried unsetting anchor tag properties but hamburger is also picking up properties from body tag of styles.less.
> > Will discuss this again in standup.
> 
> Would using `all: unset` or doing something similar in the copied over clientlibs styles at the root level help? See https://developer.mozilla.org/en-US/docs/Web/CSS/all

Tried for body but it not worked there, working fine for anchor tag. But somehow i have done the changes and now ui is almost same.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**